clients/admin-ui/src/features/privacy-experience/PrivacyExperiencesTable.tsx

```diff
@@ -1,130 +1,241 @@
 import JavaScriptTag from "./JavaScriptTag";
+import { useGetAllExperienceConfigsQuery } from "./privacy-experience.slice";
+
+const emptyExperienceResponse = {
+  items: [],
+  total: 0,
+  page: 1,
+  size: 25,
+  pages: 1,
+};
+
+const EmptyTableExperience = () => (
+  <VStack
+    mt={6}
+    p={10}
+    spacing={4}
+    borderRadius="base"
+    maxW="70%"
+    data-testid="empty-state"
+    alignSelf="center"
+    margin="auto"
+  >
+    <VStack>
+      <Text fontSize="md" fontWeight="600">
+        No privacy experiences found.
+      </Text>
+      <Text fontSize="sm">
+        Click &quot;Create new experience&quot; to add your first privacy
+        experience to Fides.
+      </Text>
+    </VStack>
+    <NextLink href={`${PRIVACY_EXPERIENCE_ROUTE}/new`}>
+      <Button
+        size="xs"
+        colorScheme="primary"
+        data-testid="add-privacy-experience-btn"
+      >
+        Create new experience
+      </Button>
+    </NextLink>
+  </VStack>
+);
+const columnHelper = createColumnHelper<ExperienceConfigListViewResponse>();
 
-const PrivacyExperiencesTable = () => {
+export const PrivacyExperiencesTable = () => {
+  const { isLoading: isLoadingHealthCheck } = useGetHealthQuery();
   const router = useRouter();
-  // Subscribe to get all privacy experiences
-  const page = useAppSelector(selectPage);
-  const pageSize = useAppSelector(selectPageSize);
-  const { isLoading } = useGetAllExperienceConfigsQuery({
-    page,
-    size: pageSize,
-  });
-  const privacyExperiences = useAppSelector(selectAllExperienceConfigs);
+
   // Permissions
   const userCanUpdate = useHasPermission([
     ScopeRegistryEnum.PRIVACY_EXPERIENCE_UPDATE,
   ]);
 
-  const handleRowClick = ({ id }: ExperienceConfigListViewResponse) => {
+  const {
+    PAGE_SIZES,
+    pageSize,
+    setPageSize,
+    onPreviousPageClick,
+    isPreviousPageDisabled,
+    onNextPageClick,
+    isNextPageDisabled,
+    startRange,
+    endRange,
+    pageIndex,
+    setTotalPages,
+  } = useServerSidePagination();
+
+  const {
+    isFetching,
+    isLoading,
+    data: experiences,
+  } = useGetAllExperienceConfigsQuery({
+    page: pageIndex,
+    size: pageSize,
+  });
+
+  const {
+    items: data,
+    total: totalRows,
+    pages: totalPages,
+  } = useMemo(() => experiences || emptyExperienceResponse, [experiences]);
+
+  useEffect(() => {
+    setTotalPages(totalPages);
+  }, [totalPages, setTotalPages]);
+
+  const inventoryColumns: ColumnDef<ExperienceConfigListViewResponse, any>[] =
+    useMemo(
+      () =>
+        [
+          columnHelper.accessor((row) => row.name, {
+            id: "name",
+            cell: (props) => <DefaultCell value={props.getValue()} />,
+            header: (props) => <DefaultHeaderCell value="Title" {...props} />,
+          }),
+          columnHelper.accessor((row) => row.component, {
+            id: "component",
+            cell: (props) => ComponentCell(props.getValue()),
```

Comment:
I think a `DefaultCell` would be more in-line with what existed here before.
```
export const ComponentCell = (value: ComponentType) => {
  const innerText = COMPONENT_MAP.get(value) ?? value;
  return <DefaultCell value={innerText} />;
};
```
![image](https://github.com/ethyca/fides/assets/2092272/2d1a2e78-5991-4eca-93b4-88f53f9a5b90)